### ISSUE TITLE
Consider nudging users towards using f-strings

### ISSUE THREAD

**Issue by NeilGirdhar** [PyCQA/pylint#3592]
Python 3.5 is end-of-life in September 2020.  Since Python 3.6 introduced f-strings, there is no good reason to interpolate _literal strings_ with the string interpolation operator
```python
'Python %f' % version
```
or the `format` method
```python
'Python {}'.format(version)
```
—except when the arguments involve unpacking:
```python
'Python {} or {}`.format(*versions)
```

Can we consider a pylint warning to nudge users towards using f-strings when possible?  It would be extremely helpful to write the f-string for them.  For example:
```
a.py:1:0: W999: Not using f-string: f"Python {version}" (not-using-fstring)
```
What do you think?

**Comment by TylerYep** [comment]
Seconded, especially since f-strings are a lot faster in practice as well.

**Comment by Pierre-Sassoulas** [comment]
It seems reasonable to me. I just think that the message should be a convention instead of a warning.
